README.md

```diff
@@ -82,48 +67,94 @@ cmake -DCMAKE_BUILD_TYPE=Release ..
 make
 ```
 
-To test everything went all right:
+To install in directory `/X`, with libraries going to `X/lib`, `Python` modules to
+`X/lib/pythonA.B/site-packages/sopt`, etc, do:
 
 ```
 cd /path/to/code/build
-ctest .
+cmake -DCMAKE_INSTALL_PREFIX=/X ..
+make install
 ```
 
-To install in directory `/X`, with libraries going to `X/lib`, python modules to
-`X/lib/pythonA.B/site-packages/sopt`, etc, do:
+Testing
+-------
+
+To check everything went all right, run the test suite:
 
 ```
 cd /path/to/code/build
-cmake -DCMAKE_INSTALL_PREFIX=/X ..
-make install
+ctest .
 ```
 
+Contributors
+------------
+
+**SOPT** was initially created by Rafael Carrillo, Jason McEwen and Yves Wiaux but major contributions
+have since been made by a number of others. The full list of contributors is as follows:
+
+* [Rafael E. Carrillo](https://www.researchgate.net/profile/Rafael_Carrillo2)
+* [Jason D. McEwen](http://www.jasonmcewen.org)
+* [Yves Wiaux](http://basp.eps.hw.ac.uk)
+* [Vijay Kartik](https://people.epfl.ch/vijay.kartik)
+* [Mayeul d'Avezac](https://github.com/mdavezac)
+* [Luke Pratley](https://about.me/luke.pratley)
+* [David Perez-Suarez](https://dpshelio.github.io)
+* [Ilektra Christidi](https://github.com/ilectra)
+* [Roland Guichard](https://github.com/UCLGuichard)
+
```

Comment:
I think Jason said to put the authors in alphabetical order?

Reply:
Yes, for the next round of reformatting.